flutter/realm_flutter/tests/linux/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 # Project-level configuration.
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.10)
 project(runner LANGUAGES CXX)
 
 # The name of the executable created for the application. Change this to change
 # the on-disk name of your application.
 set(BINARY_NAME "tests")
 # The unique GTK application identifier for this application. See:
 # https://wiki.gnome.org/HowDoI/ChooseApplicationID
-set(APPLICATION_ID "io.realm.tests")
+set(APPLICATION_ID "com.example.tests")
```

Comment:
Same here - I guess it doesn't really matter, but should we use a more branded id?

Reply:
Forgot to add `--org` flag when I re-created the platform projects. I have returned to using `io.realm`.